src/Components/Share/ShareButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const ShareButton = ({id}) => {
```

Comment:
`id` exists as a parameter in case we ever want to have multiple share options, in which case each would get it's own ID (the button + modal must share a common id).  This also allows multiple triggers for the same modal (sharing the same ID).